test/pdo_sqlsrv/pdo_270_fetch_binary.phpt

```diff
@@ -0,0 +1,95 @@
+--TEST--
+Test fetch from binary, varbinary, varbinary(max), image columns, without setting binary encoding.
+--DESCRIPTION--
+Verifies GitHub issue 270 is fixed, users could not retrieve the data as inserted in binary columns without setting the binary encoding either on stmt or using bindCoulmn encoding.
+This test versifies that the data inserted in binary columns can be retrieved using fetch, fetchColumn, fetchObject, and fetchALL functions.
+
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+$tableName = 'test_binary';
+$columns = array( 'col1', 'col2', 'col3', 'col4');
+
+// Connect
+$conn = new PDO( "sqlsrv:server=$serverName", "$username", "$password" );
+
+// CREATE database
+$conn->query("CREATE DATABASE ". $dbName) ?: die();
```

Comment:
it is better to have self-contained exportable tests, and since the db is dropped at the end of the test, doesn't matter if the table name is unique or not